woocommerce/woocommerce
# Refetch "Installed extensions" card data after installing recommended channels

### All Submissions:


### Changes proposed in this Pull Request:



Closes https://github.com/woocommerce/woocommerce/issues/37127.

In this PR, after we install and activate a recommended marketing channel that has not registered itself as a marketing channel, we refetch the data for the "Installed extensions" card so that the extension will show up in the card without needing to reload the page.

Demo video:

https://user-images.githubusercontent.com/417342/225971052-1715c550-3c19-4f46-a206-a92ad99ad304.mp4



### How to test the changes in this Pull Request:




Pre-condition: 

- Enable WooCommerce Multichannel Marketing experience in WooCommerce Settings > Advanced > Features (`/wp-admin/admin.php?page=wc-settings&tab=advanced&section=features`).
- We will use Pinterest extension for testing here. You should not have it installed.

Test instructions:

1. Go to Marketing page at `/wp-admin/admin.php?page=wc-admin&path=%2Fmarketing`.
2. In the Channels card, click on the "Add channels" toggle button. You should see Pinterest as a recommended channel.
3. Click on the "Install plugin" button for Pinterest. 
4. When plugin installation and activation is done, you should see Pinterest show up in the "Installed extensions" card.



### Other information:




### FOR PR REVIEWER ONLY:
